src/runtime_src/xdp/profile/database/static_info/xclbin_info.cpp

```diff
@@ -94,17 +145,568 @@ namespace xdp {
     }
   }
 
+  AIEInfo& AIEInfo::operator=(const AIEInfo& other)
```

Comment:
Using "source"/"src" as the name of the source AIEInfo, in place of "other" is more intuitive and readable.